racket/src/racket/cmdline.inc

```diff
@@ -672,7 +672,7 @@ static int finish_cmd_line_run(FinishArgs *fa, Repl_Proc repl)
     Scheme_Object *a[2];
     a[0] = get_init_filename(fa->global_env, INIT_FILENAME_CONF_SYM, DEFAULT_INIT_MODULE);
     if (a[0]) {
-      a[1] = scheme_false;
+      a[1] = scheme_make_integer(0);
```

Comment:
Yes it was, outside of the technical difference the docs give, I still don't have a good intuition when to use 0 or #f. Anyway, `scheme_make_false()` works. Do you know why the literal `scheme_false` doesn't work on windows?